hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OmKeysDeleteRequestWithFSO.java

```diff
@@ -42,6 +46,8 @@
  */
 public class OmKeysDeleteRequestWithFSO extends OMKeysDeleteRequest {
 
+  public static final Logger LOG = LoggerFactory.getLogger(OmKeysDeleteRequestWithFSO.class);
+
   public OmKeysDeleteRequestWithFSO(
```

Comment:
interesting that this class is called 'OmKeysDeleteRequestWithFSO' not 'OMKeysDeleteRequestWithFSO' but fine.